test/VspSpec.hs

```diff
@@ -245,3 +248,15 @@ vspSpec = do
           , "baz ="
           , "  putStrLn \"hello world\""
           ]
+    it "indexes using utf-16 codepoints" $ do
+      let
+        orig = unlines
+          [ "a𐐀b"
+          , "a𐐀b"
+          ]
+        new = applyChange (Yi.fromString orig)
+                $ J.TextDocumentContentChangeEvent (mkRange 1 0 1 3) (Just 3) "𐐀𐐀"
+      lines (Yi.toString new) `shouldBe`
```

Comment:
Yi.toString new

Does this compile?

Reply:
Thanks, I must have messed up a rebase. Fixed!